rio_tiler/colormap.py

```diff
@@ -74,19 +73,17 @@ def make_lut(colormap: Dict) -> numpy.ndarray:
     """
     lut = numpy.zeros(shape=(256, 4), dtype=numpy.uint8)
     for i, color in colormap.items():
-        lut[int(i)] = color
+        lut[int(i)] = numpy.array(color)
```

Comment:
not necessary but I think it's help understanding that we put the tuple in the numpy array